### ISSUE TITLE
Error when used with asyncio.Queue and unittest.IsolatedAsyncioTestCase

### ISSUE THREAD

**Issue by Fidget-Spinner** [dmippolitov/pydnsbl#16]
When running with unittest.IsolatedAsyncioTestCase and feeding async_check tasks into an asyncio.Queue to be consumed, the following error occurs:
```python
Traceback (most recent call last):
  File "\lib\unittest\async_case.py", line 65, in _callTestMethod
    self._callMaybeAsync(method)
  File "\lib\unittest\async_case.py", line 88, in _callMaybeAsync
    return self._asyncioTestLoop.run_until_complete(fut)
  File "\lib\asyncio\base_events.py", line 616, in run_until_complete
    return future.result()
  File "\lib\unittest\async_case.py", line 102, in _asyncioLoopRunner
    ret = await awaitable
  ...
  File "\lib\site-packages\pydnsbl\checker.py", line 140, in check_async
    results = await asyncio.gather(*tasks)
  File "\lib\site-packages\pydnsbl\checker.py", line 122, in dnsbl_request
    response = await self._resolver.query(dnsbl_query, 'A')
RuntimeError: Task <Task pending name='Task-2' coro=<dnsbl_request() running at \lib\site-packages\pydnsbl\checker.py:122> cb=[gather.<locals>._done_callback() at \lib\asyncio\tasks.py:751] created at \lib\asyncio\tasks.py:806> got Future <Future pending> attached to a different loop
```
This happens when not manually creating and passing a loop into `BaseDNSBLChecker`: `BaseDNSBLChecker` will then create and set a new loop in the main thread, even if an existing loop is already running.

The fix for this is to change the code in `BaseDNSBLChecker` (line 95 of file):
```python
        if not loop:
            self._loop = asyncio.new_event_loop()
            asyncio.set_event_loop(self._loop)
```
to this:
```python
        if not loop:
            self._loop = asyncio.get_event_loop()
            asyncio.set_event_loop(self._loop)
```

**Comment by dmippolitov** [comment]
Should be fixed in 1.1.2

**Comment by Fidget-Spinner** [comment]
Thank you!
